USAGE_DATA.md

```diff
@@ -0,0 +1,10 @@
+## [IntelliJ Knative by Red Hat](https://github.com/redhat-developer/intellij-knative)
+
+### Usage Data
+
+* when plugin is started
+* when an action is executed
```

Comment:
The cluster info is missing (see the Kubernetes plugin)
Also I think some action records the resource kind (see the Kubernetes plugin for help)